stackslib/src/config/mod.rs

```diff
@@ -1355,106 +1361,104 @@ pub struct BurnchainConfig {
     ///
     /// The key is used to derive the Bitcoin address that receives the coinbase rewards
     /// when generating blocks on the regtest network.
-    ///
-    /// Mandatory if [`BurnchainConfig::mode`] is "helium".
-    /// This is intended strictly for testing purposes.
-    ///
-    /// Default: `None`
+    /// ---
+    /// @default: `None`
+    /// @notes:
+    ///   - Mandatory if [`BurnchainConfig::mode`] is "helium".
+    ///   - This is intended strictly for testing purposes.
     pub local_mining_public_key: Option<String>,
     /// Optional bitcoin block height at which the Stacks node process should gracefully exit.
     /// When bitcoin reaches this height, the node logs a message and initiates a graceful shutdown.
-    ///
-    /// Applied only if [`BurnchainConfig::mode`] is not "mainnet".
-    /// This is intended strictly for testing purposes.
-    ///
-    /// Default: `None`
+    /// ---
+    /// @default: `None`
+    /// @notes:
+    ///   - Applied only if [`BurnchainConfig::mode`] is not "mainnet".
+    ///   - This is intended strictly for testing purposes.
     pub process_exit_at_block_height: Option<u64>,
     /// The interval, in seconds, at which the node polls the bitcoin node for new blocks and state updates.
     ///
     /// The default value of 10 seconds is mainly intended for testing purposes.
     /// It's suggested to set this to a higher value for mainnet, e.g., 300 seconds (5 minutes).
-    ///
-    /// Default: `10` seconds
+    /// ---
+    /// @default: `10` (seconds)
```

Comment:
this seems unnecessary in this context - the `@notes` specify the unit is in seconds, i would either remove from this line, or remove the `@notes` section